code/game/mecha/mecha.dm

```diff
@@ -501,6 +501,8 @@
 
 /obj/mecha/attack_animal(mob/living/simple_animal/user as mob)
 	log_message("Attack by simple animal. Attacker - [user].",1)
+	user.changeNext_move(CLICK_CD_MELEE)
+	user.do_attack_animation(src)

```

Comment:
I don't believe so, we generally have to add attack cooldowns to every attack_animal proc separately. The other attack_ procs in the mecha file also have custom cooldowns (attack_alien, etc.)